README.md

```diff
@@ -85,96 +84,69 @@ public void login(String username, String password, String regTimeout) {
    endpoint.login(username, password, regTimeout);
 }
 
-//To unregister with SIP Server
-public void logout() {
-   endpoint.logout();
+//To register with SIP Server using JWT Token, device token and certificateid
+public boolean loginWithJwtToken(String token, String JWTToken, String certificateId) {
+        return endpoint.loginWithJwtToken(JWTToken, token, certificateId);
 }
-```
-
-#### With Access Tokens/ JWT
-You can register an endpoint using:
-
-- Access Token, device token, and certificate ID
-
-```
-public boolean loginWithJwtToken(String JWTToken, String deviceToken, String certificateId)
-```
-
-- Access Token, and device token
 
-```
-public boolean loginWithJwtToken(String JWTToken, String deviceToken)
-```
+//To register with SIP Server using JWT Token and device token
+public boolean loginWithJwtToken(String token, String JWTToken) {
+        return endpoint.loginWithJwtToken(JWTToken, token);
+}
 
-- Access Token
+//To register with SIP Server using JWT
+public boolean loginWithJwtToken(String JWTToken) {
+        return endpoint.loginWithJwtToken(JWTToken);
```

Comment:
we can directly write the methods. Encapsulating the actual method inside these functions might confuse developers